gapii/cc/call_observer.cpp

```diff
@@ -112,7 +113,7 @@ void CallObserver::invoke() {
 
 void CallObserver::encodeAndDeleteCommand(::google::protobuf::Message* cmd) {
     observePending();
-    auto encoder = mSpyPtr->getEncoder();
+    auto encoder = mSpyPtr->getEncoder(mApi);
```

Comment:
When we call `encodeAndDeletCommand`, what is the value of `mApi`? I think the value is `0` as in spy.cpp, the observer is constructed with `0` value. But I think the value needs to be `1` as that is the bit mask value for Core API?